tests/test_data/test_app_dir_struct/data_sources/DemoApplicationDataSource.json

```diff
@@ -14,5 +14,8 @@
         "blob"
       ]
     }
-  }
+  },
+  "global_folders": [
+    "global"
+  ]
```

Comment:
Could this be just an implicit thing?
So the user don't have to think about where this is and what it's called. Perhaps call it "_global_"